Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -545,6 +777,881 @@ def get_ip_reputation(client: Client, score_calc: DBotScoreCalculator, ip, statu
+                                                                                      ", ".join([item.get('type')
+                                                                                                for item in hdr]))})
+    if not import_id:
+        for item in outputs:
+            item['ImportID'] = item.pop('id')
+            item['JobID'] = item.pop('jobID')
+    else:
+        outputs['ImportID'] = outputs.pop('id')
+        outputs['JobID'] = outputs.pop('jobID')
+    return CommandResults(
+        outputs_prefix=f"{THREAT_STREAM}.Import",
+        outputs_key_field="ImportID",
+        ignore_auto_extract=True,
+        outputs=outputs,
+        readable_output=readable_output,
+        raw_response=res,
+    )
+
+
+def approve_import_job_command(client: Client, import_id: str = None) -> CommandResults:
+    """
+    Approving all observables in an import job
+    Args:
+        client (Client): Client to perform calls to Anomali ThreatStream service.
+        import_id (Str):  The id of a specific import entry.
+    Returns:
+        (CommandResults).
+    """
+    res = client.approve_import_job_request(import_id)
+    if res.get("approved_by_id"):
+        readable_output = 'The import session was successfully approved.'
+    else:
+        raise DemistoException('Import Session Approval Failed.')
+    return CommandResults(
+        readable_output=readable_output,
+        raw_response=res,
+    )
+
+
+def validate_values_search_threat_model(model_type: str = None, publication_status: str = None,
+                                        signature_type: str = None):
+    """
+    Validates arguments values in search-threat-model command.
+    Args:
+        model_type (Str): A comma-separated list of model types. Supported values are: actor, attackpattern , campaign,
+        courseofaction, incident,identity, infrastructure, intrusionset, malware,signature, tipreport, ttp, tool, vulnerability.
+        publication_status (Str): A comma-separated list of publication status. Supported values are: new, pending_review,
+        review_requested, reviewed, published statuses.
+        signature_type (Str): A comma-separated list of signature type. Supported values are: Bro, Carbon Black Query,
+        ClamAV, Custom, CybOX, OpenIOC, RSA NetWitness, Snort, Splunk Query, Suricata, YARA.
+    Returns:
+        (DemistoException).
+    """
+    model_type_list = argToList(model_type)
+    publication_status_list = argToList(publication_status)
+    signature_type_list = argToList(signature_type)
+    for model_type_item in model_type_list:
+        if model_type_item.lower() not in ['actor', 'attackpattern', 'campaign', 'courseofaction', 'incident',
+                                           'identity', 'infrastructure', 'intrusionset',
+                                           'malware', 'signature', 'tipreport', 'ttp', 'tool', 'vulnerability']:
+            raise DemistoException('Unvalid values model_type argument')
```

Comment:
Also add to the validation which one is invalid
Should be something like:
f'The following model_type argument is invalid - {model_type_item}'